drivers/note/Make.defs

```diff
@@ -19,11 +19,10 @@
 ############################################################################
 
 ifeq ($(CONFIG_DRIVER_NOTE),y)
-  ifeq ($(CONFIG_SCHED_INSTRUMENTATION_EXTERNAL),)
```

Comment:
since not all drivers convert to the new style, SCHED_INSTRUMENTATION_EXTERNAL need keep as before.